adafruit_sharpmemorydisplay.py

```diff
@@ -135,7 +138,7 @@ def image(self, img):
                 self.buf[i] = 0
             # Iterate through the pixels
             for x in range(width):  # yes this double loop is slow,
-                for y in range(height):  #  but these displays are small!
+                for y in range(height):  # but these displays are small!
```

Comment:
Yeah, it's just that the CI will continue to complain.  If you want to be extra sure, [you can run pre-commit yourself](https://learn.adafruit.com/creating-and-sharing-a-circuitpython-library/check-your-code), and let it decide what how to format everything.  Pre-commit is what the CI uses, so that's the surest way.

Reply:
`black` is a formatter, and our CI tool `pre-commit` uses it to standardize the formatting of our code.